### ISSUE TITLE
Reflection warnings in NDArray

### ISSUE THREAD

**Issue by mikera** [mikera/core.matrix#47]
The current NDArray implementation has some reflection warnings whenever `mvn test` is run:

```
Reflection warning, clojure/core/matrix/impl/ndarray.clj:123 - reference to field data can't be resolved.
Reflection warning, clojure/core/matrix/impl/ndarray.clj:124 - reference to field ndims can't be resolved.
Reflection warning, clojure/core/matrix/impl/ndarray.clj:125 - reference to field shape can't be resolved.
Reflection warning, clojure/core/matrix/impl/ndarray.clj:126 - reference to field strides can't be resolved.
Reflection warning, clojure/core/matrix/impl/ndarray.clj:127 - reference to field offset can't be resolved.
```

**Comment by si14** [comment]
Fixed in 2671442.

**Comment by mikera** [comment]
cool looks good!
